src/components/Button.jsx

```diff
@@ -1,3 +1,15 @@
-const Button = () => {};
+import Link from "next/link";
 
+const Button = ({ text }) => {
+  return (
+    <>
+      <Link
+        href="https://example.com/"
+        className="inline-block bg-biscuits-purple-100 text-black font-inika py-2.4 px-8 rounded-full font-bold text-lg duration-300 hover:opacity-80"
```

Comment:
overall structure looks good! just make sure colors match the figma and you'll be good.